pulp-openshift-template.yaml

```diff
@@ -0,0 +1,1001 @@
+          resources: {}
+          securityContext:
+            privileged: true
+            runAsUser: 0
+          terminationMessagePath: /dev/termination-log
+          volumeMounts:
+          - mountPath: /dev/log
+            name: dev-log-volume
+          - mountPath: /etc/pulp
+            name: etc-pulp-volume
+          - mountPath: /etc/pki/pulp
+            name: etc-pki-pulp-volume
+          - mountPath: /var/lib/pulp
+            name: var-lib-pulp-volume
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        securityContext: {}
+        terminationGracePeriodSeconds: 30
+        volumes:
+        - name: dev-log-volume
+          persistentVolumeClaim:
+            claimName: dev-log-claim
+        - name: etc-pulp-volume
+          persistentVolumeClaim:
+            claimName: etc-pulp-claim
+        - name: etc-pki-pulp-volume
+          persistentVolumeClaim:
+            claimName: etc-pki-pulp-claim
+        - name: var-lib-pulp-volume
+          persistentVolumeClaim:
+            claimName: var-lib-pulp-claim
+    test: false
+    triggers:
+    - type: ConfigChange
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - worker
+        from:
+          kind: ImageStreamTag
+          name: worker:latest
+          namespace: pulp
+      type: ImageChange
+  status: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: null
+    name: crane
+  spec:
+    ports:
+    - port: 5000
+      protocol: TCP
+      targetPort: 80
+    selector:
+      name: crane
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: null
+    name: db
+  spec:
+    ports:
+    - port: 27017
+      protocol: TCP
+      targetPort: 27017
+    selector:
+      name: mongodb
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: null
+    name: pulpapi
+  spec:
+    ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 80
+    - name: https
+      port: 443
+      protocol: TCP
+      targetPort: 443
+    selector:
+      name: pulpapi
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: null
+    name: qpid
+  spec:
+    ports:
+    - port: 5672
+      protocol: TCP
+      targetPort: 5672
+    selector:
+      name: qpid
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    name: prepare
+    annotations:
+      openshift.io/scc: privileged
+  spec:
+    containers:
+    - name: prepare
+      image: base
+      imagePullPolicy: Always
+      command: ["bash",  "-c", "/setup.sh"]
+      env:
+        - name: QPID
+          value: qpid.${NAMESPACE}.svc.cluster.local
```

Comment:
Weird, that didn't work for me, but maybe SDN was not installed on a test cluster